pages/common/indent.md

```diff
@@ -0,0 +1,16 @@
+# indent
+
+> Change the appearance of a C/C++ program by inserting or deleting whitespace.
+> More information: <https://www.gnu.org/software/indent/>.
+
+- Format C/C++ source according to the Linux style guide:
+
+`indent -linux {{source1.c}} > {{source1.out}}`
+
+- Format C/C++ source according to the GNU style:
+
+`indent -gnu {{source2.c}} > {{source2.out}}`
```

Comment:
The docs says this command needs the `--standard-output` argument to write to `stdout`, or the `--output-file` (`-o`) to don't replace the original file.
https://www.gnu.org/software/indent/manual/indent.html#IDX3

```suggestion
`indent --gnu-style {{path/to/source.c}} > {{path/to/indented_source.c}}`
```

Reply:
Corrected the examples to use -o argument.